Base.rte/AI/HumanBehaviors.lua

```diff
@@ -2001,7 +2007,11 @@ function HumanBehaviors.GoToWpt(AI, Owner, Abort)
 		if (AI.Target and AI.BehaviorName ~= "AttackTarget") or
 			(Owner.AIMode ~= Actor.AIMODE_SQUAD and (AI.BehaviorName == "ShootArea" or AI.BehaviorName == "FaceAlarm"))
 		then
-			AI.lateralMoveState = Actor.LAT_STILL
+			if AI.behaviorType == 1 then	-- aggressive (4zK)
```

Comment:
The tags were originally for me to quickly navigate to my own parts in the script, I'll remove them now.

Reply:
oh sorry, behaviour type all done in lua? You still can actually, and it'll look better, just make a table like
`AI.behaviourType = {BEHAVIOUR_TYPE_1 = 1, BEHAVIOUR_TYPE_2 = 2, etc.}`
and you can reference them with `AI.behaviourType.BEHAVIOUR_TYPE_1`. That's pretty much all these enums are in cpp actually.

Reply:
Cool! Yeah, it's my custom prototype. The aggressive type basically makes the actor pursue the waypoint even when engaging an enemy.

Reply:
Ahhh, gotcha, it's been a while since I've looked at any of this stuff and I got mixed up, sorry. Neat stuff though!

And yeah, you can mimic enums with tables (and I'd recommend it honestly) so it makes it clearer to read. I'll leave it up to you if you wanna use ENUM_CASING or just go with normal camelCasing; for something like this I think both options are valid, as long as we remain consistent.